manifests/instance.pp

```diff
@@ -18,6 +18,8 @@
 #   enable service (boolean)
 # [*systemd_service_type*]
 #   value for systemd service type
+# [*systemd_workdir*]
```

Comment:
I would prefer to keep extra systemd options outside the scope of this module, and let the user create a systemd drop-in if he/she needs to add a specific option, otherwise the number of parameters will rapidly get out of control.